game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -28,19 +31,30 @@ class GameThread(scoreThread: ScoreThread, teamIDs: List[Int]) extends Runnable
     states get gameId
   }
 
-  def createRankedGame = {
+  def createRankedGame: Game = {
```

Comment:
Great, let's make sure to refactor this and its tests into a `createGame` or similar when we implement private games!